src/Mod/Draft/DraftGeomUtils.py

```diff
@@ -1126,6 +1126,15 @@ def isReallyClosed(wire):
     # aboveWire.isClosed() == True, but Wire[-1] are the 3rd vertex for the rectangle
     # - use Edges[i].Vertexes[0/1] instead
     length = len(wire.Edges)
+
+    # Test if it is full circle / ellipse first
+    if length == 1:
+        if len(wire.Edges[0].Vertexes) == 1:
+            return True # This is a closed wire - full circle / ellipse
```

Comment:
I would add an explanation here.
```py
if len(wire.Edges[0].Vertexes) == 1:
    # The shape has only one edge, and only one vertex, the center.
    # Therefore, it is a circle or ellipse.
    return True
else:
    # It has more than one vertex. It is not not a circle nor ellipse.
    return False
```

Why do you add TODO? TODO is something that needs to be done, it is not a generic marker to draw attention. Don't use it this way. Use it only when some implementation is missing and needs to be done.

Reply:
Thanks for comment.  
There is a remark in TODO reminding myself something I am not sure at the moment and would like to explore more to ascertain the code is correct before remove it.